sys/universal_address/universal_address.c

```diff
@@ -28,7 +34,17 @@
 /**
  * @brief Maximum number of entries handled
  */
-#define UNIVERSAL_ADDRESS_MAX_ENTRIES (40)
+/* determine the maximum numer of entries */
+#ifndef UNIVERSAL_ADDRESS_MAX_ENTRIES
```

Comment:
In general, sounds like a good idea, but for now I would leave it like this and change it once there is a need for this.